src/community/ogcapi/dggs/ogcapi-dggs/src/test/java/org/geoserver/ogcapi/v1/dggs/DGGSServiceDescriptionProviderTest.java

```diff
@@ -0,0 +1,57 @@
+/* (c) 2024 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.ogcapi.v1.dggs;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import org.geoserver.test.GeoServerSystemTestSupport;
+import org.junit.Test;
+
+/** Tests DGGSServiceDescriptionProvider and (indirectly) OgcApiServiceDescriptionProvider */
+public class DGGSServiceDescriptionProviderTest extends GeoServerSystemTestSupport {
+
+    /** basic test of the description* */
+    @Test
+    public void testProvider() {
+        var provider = new DGGSServiceDescriptionProvider(this.getGeoServer());
+
+        var descriptions = provider.getServices(null, null);
+        assertEquals(1, descriptions.size());
+        assertEquals("WMTS", descriptions.get(0).getServiceType());
+        assertTrue(descriptions.get(0).getDescriptionPriority() < 100);
+    }
+
+    /** tests the links - esp the actual link when its WS-based of WS-and-layer based* */
+    @Test
+    public void testLinks() {
+        var provider = new DGGSServiceDescriptionProvider(this.getGeoServer());
+
+        var links = provider.getServiceLinks(null, null);
+        assertEquals(1, links.size());
+        assertEquals("WMTS", links.get(0).getServiceType());
+        assertEquals("DGGS", links.get(0).getSpecificServiceType());
```

Comment:
Q: Is SpecificServiceType still needed now that Andrea has moved DGGS into its own heading (not part of WMTS?)

Reply:
I think it is still needed - there's can be more services added later.  I think its needed for the grouped ones (like Features).